src/functions/module_custom.ts

```diff
@@ -9,10 +9,10 @@ import { toNumber } from "./helpers";
 // -----------------------------------------------------------------------------
 export const FORMAT_LARGE_NUMBER: AddFunctionDescription = {
   description: _lt(`Apply a large number format`),
-  args: args(`
-      value (number) ${_lt("The number.")}
-      unit (string, optional) ${_lt("The formatting unit. Use 'k', 'm', or 'b' to force the unit")}
-    `),
+  args: args(
+    ["value (number)", _lt("The number.")],
+    ["unit (string, optional)", _lt("The formatting unit. Use 'k', 'm', or 'b' to force the unit")]
+  ),
```

Comment:
Alternative way of writing the same thing.
Not sure it's better. Maybe it feels less weird than an arbitrary array with 2 elements 
WDYT?
```suggestion
  args: [
    arg("value (number)", _lt("The number.")),
    arg("unit (string, optional)", _lt("The formatting unit. Use 'k', 'm', or 'b' to force the unit"))
  ],
```

Reply:
As it's a code that is not often read, I think it will be easier to go and see the definition of `arg` instead of the function `args` to see what we do with the array.
Let's go for this solution